### ISSUE TITLE
With bookmarks corner tap disabled, the upper right is non-tappable

### ISSUE THREAD

**Issue by chucktomasi** [mb1986/rm-hacks#179]
I'm using  0.0.8 with 3.9.3.1986 and disabled Bookmarks Corner Tap. I was hoping the icon would just disappear, but when using something like recalendar.me, that arrow to move forward a month no longer works. Seems a bit overkill. Thx

**Comment by mb1986** [comment]
Could you please verify if it is better on 0.0.9-alpha.6:

```sh
sh -c "$(wget https://raw.githubusercontent.com/mb1986/rm-hacks/main/install.sh -O-)" _ patch 0.0.9-alpha.6
```

**Comment by chucktomasi** [comment]
The issue with the bookmarks tap region in the upper right seems better, but I noticed a new issue with alpha 6. E.g. After systemctl restart xochitl.service, the keyboard appears by default when I open a doc - particularly annoying on my PDF as there's no "T" icon. If I open a notebook, then close, and go back to the PDF, it seems fine. It's only an issue after a restart if I open a PDF first. Opening a notebook first is not an issue. Feels like something didn't get initialized properly (or a default value?) LMK if you would like a different issue on that one.

**Comment by pragmatically-dev** [comment]
I had the same issue, close the open document and restart the rm2 it solved the problem for me and I couldn’t reproduce again

**Comment by rudy0ne** [comment]
The same happened to me after restarting. I thought nothing of it since it disappeared once I switched from the PDF to a notebook.

**Comment by Tridens92** [comment]
If "bookmarks corner tap" is disabled, should we still be seeing bookmarks in the corner on those pages that have them?  I would think not (since they're displayed in the toolbar).

**Comment by PepikVaio** [comment]
I had the same request but in the end it's OK - at least you can see it when the panel is closed.

**Comment by mb1986** [comment]
https://github.com/mb1986/rm-hacks/issues/179#issuecomment-1897428282 should be fixed in:

```sh
sh -c "$(wget https://raw.githubusercontent.com/mb1986/rm-hacks/main/install.sh -O-)" _ patch 0.0.9-alpha.8
```

**Comment by atngames** [comment]
Closing this issue since it is fixed in 0.0.9
